codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/credentialresolverservices/IdentityProvidingSTSClientIntegration.kt

```diff
@@ -0,0 +1,100 @@
+package software.amazon.smithy.aws.swift.codegen.customization.credentialresolverservices
```

Comment:
Customization to generate `IdentityProvidingSTSClient` struct under `InternalAWSSTS`.